core/src/main/java/com/amplifyframework/util/FieldFinder.java

```diff
@@ -17,15 +17,16 @@
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.core.util.Predicate;
```

Comment:
We have various functional interfaces like this defined in our own library, I think. Should we use AndroidX's? Or one of ours?